src/SourceLink.Tools/Microsoft.SourceLink.Tools.Package.csproj

```diff
@@ -6,6 +6,7 @@
     <GenerateDocumentationFile>false</GenerateDocumentationFile>
     <DebugType>none</DebugType>
     <GenerateDependencyFile>false</GenerateDependencyFile>
+    <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings> <!-- For System.Text.Json 8.x.y reference -->
```

Comment:
Saw other repositories do this, not sure if this is the ideal solution here @MichaelSimons

Reply:
Please add a comment describing what the purpose of the property is.

Reply:
This property shouldn't be set without a VERY strong reason. It's a hack to workaround a repository targeting a lower .NETCoreApp TFM than what the referenced packages actually support.

Reply:
```suggestion
```

Reply:
If you want to reference the live version of System.Text.Json then this project must be updated to net8.0 as source-build packages only include the latest .NETCoreApp TFM: net8.0. Falling back to the netstandard2.0 TFM isn't a great workaround either as that behaves differently than the net8.0 asset.

Reply:
Removed since the projects now utilize `$(NetCurrent)` (`net8.0`)